docs/samples/v1beta1/transformer/collocation/collocation.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: custom-transformer-collocation
+spec:
+  predictor:
+    containers:
+      - name: kserve-container
+        image: kserve/custom-model-grpc:latest
+        args:
+          - --model_name=custom-model
+          - --protocol=grpc-v2
+          - --grpc_port=8082
```

Comment:
for gRPC I think we need to expose the port with
```yaml
ports:
   - containerPort: 8082
      name: h2c
      protocol: TCP
```

Reply:
Knative does not allow setting ports for more than one container. So predictor cannot be exposed.

Reply:
Actually you do not need to expose the port as it is local networking call

Reply:
Exactly!